src/core/libraries/random/random.cpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "common/logging/log.h"
+#include "core/libraries/error_codes.h"
+#include "core/libraries/libs.h"
+#include "random.h"
+
+namespace Libraries::Random {
+
+s32 PS4_SYSV_ABI sceRandomGetRandomNumber(void* buf, size_t size) {
```

Comment:
```suggestion
s32 PS4_SYSV_ABI sceRandomGetRandomNumber(u8* buf, size_t size) {
```
No need for the cast

Reply:
fixed